src/components/organisms/BannerAdmin/ConfirmationBannerModal/ConfirmationBannerModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+
+import { Button } from "components/atoms/Button";
+
+import "./ConfirmationBannerModal.scss";
+
+interface ConfirmationBannerModalProps {
+  show?: boolean;
+  onConfirm: () => void;
+  onCancel?: () => void;
+}
+
+export const ConfirmationBannerModal: React.FC<ConfirmationBannerModalProps> = ({
```

Comment:
I'm almost certain we have an existing generic component for exactly this purpose?

Reply:
`src/components/atoms/ConfirmationModal/ConfirmationModal.tsx`

Why are we duplicating this here? I'm guessing to change the labels on it, but that's not a good reason. Just make the existing component's labels customisable + leave the existing as the defaults.

Reply:
got rid of `ConfirmationBannerModal.tsx` , extended `ConfirmationModal.tsx` 771b70a